packages/sdk/src/evm/core/classes/contract-publisher.ts

```diff
@@ -357,11 +358,33 @@ export class ContractPublisher extends RPCConnectionHandler {
       invariant(signer, "A signer is required");
       const publisher = await signer.getAddress();
 
-      const predeployMetadata = await fetchRawPredeployMetadata(
+      const predeployMetadata = await fetchPreDeployMetadata(
```

Comment:
be careful with this, i remember having a good reason for keeping the `raw` metadata around here, lets keep it as it was